system_tests/features/steps/get_info_endpoints.py

```diff
@@ -0,0 +1,89 @@
+import requests
+
+from config import Config
+from behave import given, when
+
+
+@given('the ${service} service is running')
+def service_running():
+    pass
+
+
+@when('the system requests action endpoint info')
+def requests_action_endpoint_info(context):
+    context.response = requests.get(Config.ACTION_SERVICE + Config.INFO)
+
+
+@when('the system requests action exporter endpoint info')
+def requests_action_exporter_endpoint_info(context):
+    context.response = requests.get(Config.ACTION_EXPORTER + Config.INFO)
+
+
+@when('the system requests backstage endpoint info')
+def requests_backstage_endpoint_info(context):
+    context.response = requests.get(Config.BACKSTAGE_SERVICE + Config.INFO)
+
+
+@when('the system requests case endpoint info')
+def requests_case_endpoint_info(context):
+    context.response = requests.get(Config.CASE_SERVICE + Config.INFO)
+
+
+@when('the system requests collection exercise endpoint info')
+def requests_collection_exercise_endpoint_info(context):
+    context.response = requests.get(Config.COLLECTION_EXERCISE + Config.INFO)
+
+
+@when('the system requests collection instrument endpoint info')
+def requests_collection_instrument_endpoint_info(context):
+    context.response = requests.get(Config.COLLECTION_INSTRUMENT_SERVICE + Config.INFO)
+
+
+@when('the system requests django endpoint info')
+def requests_django_endpoint_info(context):
+    context.response = requests.get(Config.DJANGO_SERVICE + Config.INFO)
+
+
+@when('the system requests frontstage-api endpoint info')
+def requests_frontstage_api_endpoint_info(context):
+    context.response = requests.get(Config.FRONTSTAGE_API_SERVICE + Config.INFO)
+
+
+@when('the system requests frontstage endpoint info')
+def requests_frontstage_endpoint_info(context):
+    context.response = requests.get(Config.FRONTSTAGE_SERVICE + Config.INFO)
+
+
+@when('the system requests iac endpoint info')
+def requests_iac_endpoint_info(context):
+    context.response = requests.get(Config.IAC_SERVICE + Config.INFO)
+
+
+@when('the system requests notify gateway endpoint info')
+def requests_notify_gateway_endpoint_info(context):
+    context.response = requests.get(Config.NOTIFY_GATEWAY_SERVICE + Config.INFO)
+
+
+@when('the system requests party endpoint info')
+def requests_party_endpoint_info(context):
+    context.response = requests.get(Config.PARTY_SERVICE + Config.INFO)
+
+
+@when('the system requests response operations ui endpoint info')
+def requests_response_operations_ui_endpoint_info(context):
+    context.response = requests.get(Config.RESPONSE_OPERATIONS_UI + Config.INFO)
```

Comment:
Response Operations UI does not have an `info` endpoint. However changing this to the following should return status code 200
`context.response = requests.get(Config.RESPONSE_OPERATIONS_UI)`

Reply:
I think it might be ras-backstage-ui which doesn't have an `info` end point currently? I was able to hit the info end point for response-operations-ui in the int space. Could we use the above suggestion for the ras-backstage-ui instead?

Reply:
ahh.. I was thinking this is the old ROPs, which we us for RM integration tests. Agree that we don't need this change as the new ROPs has an `info` endpoint. 
We can include `ras-backstage-ui` (if needed), not sure if it will be replaced by the new ROPs as well.